### DISCUSSION TITLE
MHI2Q_US_P5145_MU1316 No patch file

### DISCUSSION THREAD

**Discussion by bobrien-ssa** [Mr-MIBonk/M.I.B._More-Incredible-Bash#116] [General]
Hi,
My current firmware is MHI2Q_US_AUG22_P5145_MU1316, only MHI2Q_US_AUG22_P5087_MU1316 patch exists. Can I just rename the patch folder? Currently am uploading my backup folder to mibsolution but it is taking forever.

**Comment by bobrien-ssa** [comment]
Direct link to backup zip:

https://mibsolution.one/index.php/f/177990

**Reply by harman-f** [reply]
Yes, you can rename the folder.

ifs-root-stage2 image stayed identical

**Reply by bobrien-ssa** [reply]
Works perfect, thanks! Subwoofer control works on this firmware too!
